src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -148,6 +167,14 @@ export default {
 			default: false,
 		},
 
+		/**
+		 * If `isHeading` is set, this defines the heading level that should be used
+		 */
+		headingLevel: {
+			type: Number,
+			default: 2,
+		},
+
```

Comment:
A new feature then?

Reply:
Probably. I need to fix a place that looks like this:
```
h2
  h2 < this should be h3 but is not possible
  ul < this is a appnavigation caption
```

Reply:
Splitted the PR.